queue.tf

```diff
@@ -24,6 +25,7 @@ module "notifications-queue" {
   namespace_name      = "${module.queue-namespace.name}"
   resource_group_name = "${azurerm_resource_group.rg.name}"
   lock_duration       = "PT5M"
+  max_delivery_count  = "288"
```

Comment:
do we need to update this queue as well?

Reply:
@rkondratowicz Yeah, Nitin suggested changing this queue as well, just in case Exela notification service is down